docs/source/user/commands.rst

```diff
@@ -18,3 +29,12 @@ The command palette can be accessed from the View menu or using the keyboard sho
 ``Command/Ctrl Shift C``.
 
 The command palette can be displayed in the sidebar by adding ``'modal': false`` to the Settings.
```

Comment:
Maybe add to the doc how to get the command palette with `Cmd Shift C`

Reply:
This is the above sentence at line 17-18 :wink: